Tanks/Assets/Scripts/Shell/MineScript.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MineScript : MonoBehaviour
+{
+    public LayerMask m_TankMask;
+    public ParticleSystem m_ExplosionParticles;
+    public AudioSource m_ExplosionAudio;
+    public float m_ExplosionForce = 1000f;
+    public float m_MaxLifeTime = 30f;
+    public float m_ExplosionRadius = 5f;
```

Comment:
-If you find you're repeating something a lot, it may be worth it to abstract things out and use inheritance. You won't have to do that for this, but just keep it in mind
-Also initialize your fields and get rid of the m_